validator/node/node.go

```diff
@@ -363,6 +371,17 @@ func (s *ValidatorClient) registerClientService(
 	if err := s.services.FetchService(&sp); err == nil {
 		protector = sp
 	}
+
+	gStruct := &g.Graffiti{}
+	var err error
+	if s.cliCtx.IsSet(flags.GraffitiFileFlag.Name) {
+		n := s.cliCtx.String(flags.GraffitiFileFlag.Name)
+		gStruct, err = g.ParseGraffitiFile(n)
+		if err != nil {
+			return err
+		}
```

Comment:
Maybe just log a warning here? It would be sad if parsing a graffiti file failed and the validator remained offline

Reply:
Good point!